CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 - Renamed "testing" argument for plots to "show_plot" and flipped its meaning (show_plot=True is now the default and shows the plot) ([#3842](https://github.com/pybamm-team/PyBaMM/pull/3842))
 - Dropped support for BPX version 0.3.0 and below ([#3414](https://github.com/pybamm-team/PyBaMM/pull/3414))
+- Integrated the `[latexify]` extra into the core PyBaMM package, deprecating the `pybamm[latexify]` set of optional dependencies. SymPy is now a required dependency and will be installed upon installing PyBaMM ([#3848](https://github.com/pybamm-team/PyBaMM/pull/3848))
```

Comment:
Should go at the top

Reply:
This should be fine [`ad958a2`](https://github.com/pybamm-team/PyBaMM/pull/3848/commits/ad958a2d7dca2007a90a747a93aeade2d8206ac4), or did you mean something else?